alembic/versions/d04295661a60_003_add_new_roles.py

```diff
@@ -0,0 +1,55 @@
+"""003_add_new_roles
+
+Revision ID: d04295661a60
+Revises: d482f86e679b
+Create Date: 2020-10-08 15:37:11.639520
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd04295661a60'
+down_revision = 'd482f86e679b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add new Survey Manager role and Senior Research Officer with correct features and manage access permissions
+    # SRO role
+    sro_features = '"interviewing", "sample_upload", "case_summary", "cati_dashboard", "manage_surveys", ' \
+                   '"manage_field_periods", "manage_users"'
+    sro_manage_users = '["Research"]'
+
+    op.execute(
+        "INSERT INTO role VALUES('Senior_Research_Officer', 'Senior Research Officer', '" + sro_features + "', '" + sro_manage_users + "')")
+
+    # Survey Manager role
+    survey_manager_features = '"interviewing", "sample_upload", "case_summary", "cati_dashboard", ' \
+                              '"manage_surveys", "manage_field_periods", "manage_users"'
+    survey_manager_manage_users = '["Survey_Manager", "Research"]'
```

Comment:
Survey managers should not be able to able to manage Survey managers.
But should be able to manage Senior_Research_Officer's

```suggestion
    survey_manager_manage_users = '["Senior_Research_Officer", "Research"]'
```